### PR TITLE
Improvement for detecting nopCommerce

### PR DESCRIPTION
**Pull request by AndreiMaz**
1. Added Generator Meta
2. Added two cookies (.Nop.Customer and Nop.customer)

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -10069,9 +10069,15 @@
...
 				"6"
 			],
+      "headers": {
+ 		  	"Set-Cookie": "Nop\\.customer="
+      },
 			"html": "(?:<!--Powered by nopCommerce|Powered by: <a[^>]+nopcommerce)",
 			"icon": "nopCommerce.png",
 			"implies": "Microsoft ASP.NET",
+      "meta": {
+ 				"generator": "nopCommerce"
```
Comment (jvoisin) [inline src/apps.json]: Could it be `^noCommerce$` instead, or something like that?

Comment (AndreiMaz) [inline src/apps.json]: @jvoisin it's always "nopCommerce" (for all versions)

Comment (jvoisin) [inline src/apps.json]: Then please use `^noCommerce$` :)

Comment (AndreiMaz) [inline src/apps.json]: @jvoisin I've just made a new pull request. Although I don't udnerstand why it should be written this way if most of your other records have just names (e.g. "generator": "PrestaShop")

**Pull request closed without merging**